task-service/package.json

```diff
@@ -1,22 +1,22 @@
 {
   "private": true,
   "scripts": {
-    "start": "node ./dist/index.js",
-    "build": "rm -rf dist/ && tsc -b tsconfig.json",
-    "gcp-build": "yarn build",
-    "serve": "yarn build && yarn start",
-    "deploy": "yarn build && gcloud app deploy"
+    "build": "rm -rf dist/ && tsc -p tsconfig.json",
+    "deploy": "yarn build && gcloud functions deploy handlePresubmitPrTask --trigger-topic presubmit-pr-task --runtime nodejs8",
```

Comment:
NodeJS 8? Isn't that even older than the current LTS?

Reply:
Its the latest version available in the firebase function runtime.

The default is actually v6